### PR TITLE
GX up to 0.17+

### PR DESCRIPTION
**Pull request by VolkovGeoPhy**
### Describe your changes:
Update ingestion requirement great expectation to actual version
Fixes <issue-number>

I worked on ... because of keeping actual library versions

#
### Type of change:

- [!] Improvement

#
### Checklist:

- [!] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: ingestion/setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def get_long_description():
     "boto3": "boto3>=1.20,<2.0",  # No need to add botocore separately. It's a dep from boto3
     "geoalchemy2": "GeoAlchemy2~=0.12",
     "google-cloud-storage": "google-cloud-storage==1.43.0",
-    "great-expectations": "great-expectations~=0.16.0",
+    "great-expectations": "great-expectations>=0.17.0",
```
Comment (pmbrull) [inline ingestion/setup.py:36]: @TeddyCr should we keep this as `~=0.17`?

Comment (TeddyCr) [reply ingestion/setup.py:36]: Correct. cc @VolkovGeoPhy

**Review by TeddyCr — CHANGES_REQUESTED** [review]
We should have it as `~=0.17`

File: ingestion/setup.py (hunk 1 above)
Comment (VolkovGeoPhy) [reply ingestion/setup.py]: done

**Review by pmbrull — APPROVED** [review]

**Comment by pmbrull** [comment]
oh actually it looks like python 3.7 cannot go further than 0.6.13:

```
python version: 0.16.14 Requires-Python >=3.8;
```

Since we are not ready to deprecate python 3.7 we should hold off on this change. Created the 3.7 issue and added your PR there so that we can move on when the deprecation is done https://github.com/open-metadata/OpenMetadata/issues/12082

Thanks

**Comment by harshach** [comment]
@pmbrull since we deprecated 3.7, can we merge this in?

**Pull request merged**